dig_test.go

```diff
@@ -782,6 +782,57 @@ func TestEndToEndSuccess(t *testing.T) {
 		assert.Contains(t, err.Error(), "B isn't in the container")
 	})
 
+	t.Run("value groups", func(t *testing.T) {
```

Comment:
I like this test, but can we add a dumber one that more plainly illustrates the feature?

Reply:
This test is just a placeholder to demonstrate this feature for now.